improver_tests/blending/weights/test_WeightsUtilities.py

```diff
@@ -118,7 +120,18 @@ class Test_build_weights_cube(IrisTest):
 
     def setUp(self):
         """Setup for testing cube creation."""
-        self.cube = create_cube_with_threshold()
+        cube = set_up_probability_cube(
+            np.full((1, 2, 2), 0.5, dtype=np.float32),
+            np.array([1.0], dtype=np.float32),
+            variable_name="rainfall_rate",
+            threshold_units="mm h-1",
+            time=dt(2015, 11, 19, 1, 30),
+            frt=dt(2015, 11, 18, 22, 0),
+        )
+
+        time_points = [dt(2015, 11, 19, 0, 30), dt(2015, 11, 19, 1, 30)]
```

Comment:
Can you use the default `time` and `frt` values if you modify the `time_points` to match? It would remove the necessity of setting something unrelated to these tests.

Reply:
I'd rather not, just in case those defaults ever change.  By defining all three together I define a sensible relationship between them, in terms of forecast period.

Reply:
I don't really agree with this logic. Why would the defaults ever change? That would break loads of unit-tests. I'm not going to push for this to be changed though, it isn't important.